tools/delete_cron_outputs/README.md

```diff
@@ -0,0 +1,11 @@
+# Delete Cron Outputs
```

Comment:
very nice!

I assume this would be part of oncall task?

Reply:
Yeup, plan to add it as one of the oncall GC steps